contrib/local-environment/oauth2-proxy-alpha-config-multiple-providers.yaml

```diff
@@ -0,0 +1,50 @@
+upstreamconfig:
+  upstreams:
+    - id: httpbin
+      path: /
+      uri: http://httpbin
+injectRequestHeaders:
+  - name: X-Forwarded-Groups
+    values:
+    - claim: groups
+  - name: X-Forwarded-User
+    values:
+    - claim: user
+  - name: X-Forwarded-Email
+    values:
+    - claim: email
+  - name: X-Forwarded-Preferred-Username
+    values:
+    - claim: preferred_username
+defaultProvider: keycloak
+providers:
+- provider: oidc
+  id: dex
+  name: "@dexexample.com"
+  clientSecret: b2F1dGgyLXByb3h5LWNsaWVudC1zZWNyZXQK
+  clientID: oauth2-proxy
+  oidcConfig:
+    issuerURL: http://dex.localhost:4190/dex
+    emailClaim: email
+    groupsClaim: groups
+    insecureSkipNonce: true
+    userIDClaim: email
+    audienceClaims:
+    - aud
+- provider: oidc
+  id: keycloak
+  name: "@keycloakexample.com"
+  clientSecret: 72341b6d-7065-4518-a0e4-50ee15025608
+  clientID: oauth2-proxy
+  oidcConfig:
+    issuerURL: http://keycloak.localtest.me:9080/auth/realms/master
+    emailClaim: email
+    groupsClaim: groups
+    insecureSkipNonce: true
```

Comment:
something has happened since december, where now the oidc defaults are not set.

audienceClaims in an empty array if you do not set it atm.....